packages/apps/desktop-viewer-test/src/frontend/components/routes/HomeRoute.tsx

```diff
@@ -0,0 +1,17 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { RouteComponentProps, useNavigate } from "@reach/router";
+import React from "react";
+
+import { useMenuHandlers } from "../../hooks";
+import Home from "../home/Home";
+
+//eslint-disable-next-line no-empty-pattern
+export const HomeRoute = ({}: RouteComponentProps) => {
+  const navigate = useNavigate();
```

Comment:
should this be higher up? in App.tsx?

Reply:
specifically the useMenuHandlers hook

Reply:
It needs the router functions so needs to be in a child of the router components. I thought about that as well, but the plan is to always start the app at the home route. Once we implement defaulting to most recent, that logic (the redirect) will go in the home route (at least in my head)